docs/reference/index-modules/similarity.asciidoc

```diff
@@ -168,7 +168,7 @@ Type name: `LMJelinekMercer`
 ==== Default and Base Similarities
 
 By default, Elasticsearch will use whatever similarity is configured as
-`default`. However, the similarity functions `queryNorm()` and `coord()`
+`default` or `BM25` if default is not declared. However, the similarity functions `queryNorm()` and `coord()`
```

Comment:
Even though we currently allow to redefine the `BM25` similarity, this does not sound like a good idea to me so maybe we should not document it?